examples/exotica_examples/scripts/example_attach.py

```diff
@@ -2,15 +2,16 @@
 
 import pyexotica as exo
 from pyexotica.publish_trajectory import *
-
+from time import sleep
+import signal
 
 def handle_attaching(i, x, scene):
     # Update state
     scene.update(x)
     if i == 0:
         # Reset object pose
         scene.attach_object_local(
-            'Item', '', exo.KDLFrame([0.6, -0.3, 0.5, 0, 0, 0, 1]))
+            'Item', '', exo.KDLFrame([0.61, -0.3, 0.5, 0, 0, 0, 1]))
```

Comment:
Well, I increased Rho=1e3 so now the solver reaches exactly where it's supposed to so the it touches the box on approach. I increased the distance otherwise OMPL start/goal state are invalid.